### ISSUE TITLE
Possible for add new block not to default to text?

### ISSUE THREAD

**Issue by timmyg** [madebymany/sir-trevor-js#437]
Is there a configuration available for the functionality of the add block button? I preferred when it asked which block type you wanted immediately rather than defaulting to text and requiring another mouse click.

**Comment by raffij** [comment]
@timmyg the feeling is there'll be a toggle in sir-trevor. Block mode and document mode. Think of the current status as document mode. Now got to work back in the block mode. So this is coming.

**Comment by timmyg** [comment]
Gotcha. Sounds great!

**Comment by SgtOddball** [comment]
Oooooooooooooh now thats something to get all exited over.

**Comment by timmyg** [comment]
@raffij are there any quick hacks to default to block mode? We are launching in a few weeks and would love to have it start off in block mode.

**Comment by timmyg** [comment]
I suppose I could just figure out what happens when you click the add block plus button and then call the function that happens when you click the green plus button to the left, right?

**Comment by raffij** [comment]
@timmyg yeah that's a way to do it for now. Really need to spend some time on it. Full time on projects at the moment, so got to find time in evenings to work on it.

**Comment by raffij** [comment]
That's actaually the functionality that should happen if the you click the green cross on the left when you hover too, so something i've got to sort.
